clients/da-vinci-client/src/main/java/com/linkedin/davinci/listener/response/AdminResponse.java

```diff
@@ -65,7 +67,19 @@ public void addStoreVersionState(StoreVersionState storeVersionState) {
     try (ByteArrayOutputStream output = new ByteArrayOutputStream()) {
       GenericDatumWriter<Object> avroDatumWriter = new GenericDatumWriter<>(StoreVersionState.SCHEMA$);
       Encoder byteToHexJsonEncoder = new ByteBufferToHexFormatJsonEncoder(StoreVersionState.SCHEMA$, output);
-      avroDatumWriter.write(storeVersionState, byteToHexJsonEncoder);
+      StoreVersionState updatedState = storeVersionState;
+      if (storeVersionState.compressionDictionary != null && storeVersionState.compressionDictionary.hasRemaining()) {
+        // We don't want to dump compression dictionary since it is not readable
+        updatedState = new StoreVersionState();
+        for (Schema.Field field: StoreVersionState.SCHEMA$.getFields()) {
```

Comment:
Nitpick: IMHO it looks cleaner to use `getClassSchema()` than `SCHEMA$`.

Reply:
Fixed.